management/management_api/endpoints/endpoint_utils.py

```diff
@@ -236,3 +239,17 @@ def verify_endpoint_amount(api_instance, apps_api_instance, namespace):
         endpoint_number = get_endpoint_number(apps_api_instance, namespace)
         if endpoint_number >= int(namespace_annotations['maxEndpoints']):
             raise EndpointsReachedMaximumException()
+
+
+def check_endpoint_model(namespace, model_name):
+    try:
+        bucket = minio_resource.Bucket(name=namespace)
+    except ClientError:
+        logger.warning(f'Endpoint was created but we cant reach "{namespace}" minio bucket'

```

Comment:
@mzegla We can't throwing MinioCallException in this case,  because that Exception raise an error. In this case we cant return "Internal server error', because endpoint was created successfully and this model checker is optional feature